src/CSSBattle/36_Interleaved/HuiHui.html

```diff
@@ -0,0 +1,31 @@
+<div class="stick"></div>
+<div class="stick"></div>
+<div class="stick"></div>
+<div class="stick"></div>
+<div class="stick"></div>
+
+<style>
+  body {
+    background: #1a4341;
+    display: flex;
+    gap: 25px;
+    margin: 0 25px;
+  }
+
+  .stick {
+    width: 50px;
+    height: 300px;
+    background: #dd6b4d;
```

Comment:
Can be removed since is redeclared below

Reply:
fixed in 4f7a70a